samza-core/src/main/java/org/apache/samza/zk/ZkUtils.java

```diff
@@ -160,7 +158,7 @@ public void close() throws ZkInterruptedException {
     * @param dataListener describe this
     */
   public void subscribeToJobModelVersionChange(IZkDataListener dataListener) {
-    LOG.info("pid=" + processorId + " subscribing for jm version change at:" + keyBuilder.getJobModelVersionPath());
+    LOG.info(" subscribing for jm version change at:" + keyBuilder.getJobModelVersionPath());
```

Comment:
👍 to remove if newJobModelVersion is not used by ZkJobCoordinator. Wondering how it ends up in the code base if not used?